src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -36,6 +36,38 @@ public class RazorViewEngineOptions
         public IList<IFileProvider> FileProviders { get; } = new List<IFileProvider>();
 
         /// <summary>
+        /// Gets the locations where <see cref="RazorViewEngine"/> will search for views.
+        /// </summary>
+        /// <remarks>
+        /// The locations of the views returned from controllers that do not belong to an area.
+        /// Locations are composite format strings (see http://msdn.microsoft.com/en-us/library/txafckwd.aspx),
+        /// which contains following indexes:
+        /// {0} - Action Name
+        /// {1} - Controller Name
+        /// The values for these locations are case-sensitive on case-sensitive file systems.
+        /// For example, the view for the <c>Test</c> action of <c>HomeController</c> should be located at
+        /// <c>/Views/Home/Test.cshtml</c>. Locations such as <c>/views/home/test.cshtml</c> would not be discovered
```

Comment:
Need some `<para>` and perhaps `<ul>` love or this whole remark will run together messily.